spec/components/contents/file_component_spec.rb

```diff
@@ -3,7 +3,7 @@
 require 'rails_helper'
 
 RSpec.describe Contents::FileComponent, type: :component do
-  let(:component) { described_class.new(file: file, object_id: 'druid:kb487gt5106', viewable: true) }
+  let(:component) { described_class.new(file: file, object_id: 'druid:kb487gt5106', viewable: true, image: true) }
```

Comment:
Please add one context where image is true and one context where image is false.

Reply:
Right, I see how that would be good. Just added another spec to cover image: false. Let me know if you were thinking of something further.